lib/sitemap/sitemap_generator.rb

```diff
@@ -13,10 +14,10 @@ def self.sitemap_limit
   def get_all_documents
     Enumerator.new do |yielder|
       # Hard-code the site root, as it isn't listed in any search index
-      yielder << "/"
+      yielder << homepage_document
 
       @sitemap_indices.each do |index|
-        index.all_document_links(EXCLUDED_FORMATS).each do |document|
+        index.all_documents(exclude_formats: EXCLUDED_FORMATS).each do |document|
           if document
```

Comment:
I feel like we should be able to remove this check. We should add something to the new govuk index work to ensure the field is never blank.

@MatMoore do you agree?